### DISCUSSION TITLE
Texture evolution simulation

### DISCUSSION THREAD

**Discussion by Ming-is** [neperfepx/FEPX#66] [Q&A] [answered]
I tried to use fepx to simulate and analyze the metal texture evolution during 0.03 strain amplitude monotonic stretching, but through the visual output of the grain orientation pole diagram, it was found that the pole diagram had no obvious change, so I wondered whether fepx could be used to analyze the texture evolution. If so, what improvements should I make?

**Comment by rquey** [comment, marked as answer]
FEPX does include lattice rotation / texture evolution effects. In fact, It was first developed for this purpose. One of the first application is due to Marin and Dawson ([this paper](https://www.sciencedirect.com/science/article/abs/pii/S0045782598000346?via%3Dihub)), and a more recent one, for which FEPX was used with Neper, is [this paper](https://neper.info/docs/neper-reference-paper.pdf). See also [this paper](https://www.sciencedirect.com/science/article/abs/pii/S0022509615300363).

In your simulation, your should normally see the lattice orientations change as a result of applied deformation. However, it is certain that this evolution can only be limited, given the small strain amplitude that you used (0.03).

**Reply by mkasemer** [reply]
I will second @rquey 's comment. For many ductile metals, 0.03 strain is rather small, and we should not expect the texture to exhibit significant evolution at this point.
